infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -211,7 +211,8 @@ locals {
         GRANT_TYPE : "workload_identity_federation" # by default, assumed to be of type 'urn:ietf:params:oauth:client-assertion-type:jwt-bearer'
         TOKEN_SCOPE : "https://graph.microsoft.com/.default"
         REFRESH_ENDPOINT = "https://login.microsoftonline.com/${var.msft_tenant_id}/oauth2/v2.0/token"
-      },
+      }
+      external_todo : null
```

Comment:
@aperez-worklytics - I found I had to add this, to avoid getting errors about inconsistent types at: https://github.com/Worklytics/psoxy/pull/741/files#diff-a60bf3492962adad897238bca8f565ed1f1eb32238c04483988b29baf43a1fb2R26